controllers/dashboard-routes.js

```diff
@@ -36,14 +39,29 @@ router.get('/', withAuth, (req, res) => {
                     } else {
                         accumulator.monthlyCost += Math.round(parseFloat(current.Service.cost));
                     }
+                    activeServiceIds.push(current.Service.id)
                     accumulator.activeSubs++;
                 }
                 return accumulator;
             }, summaryData);
-            res.render('dashboard', {
-                services,
-                loggedIn: req.session.loggedIn,
-                summary: summaryData
+            TvShow.findAll({
+                where: {
+                    // use the ID from the session after testing (req.session.user_id)
+                    premiereDate: {
+                        [Op.gte]: moment()
```

Comment:
why is this changed to Op when the rest of these are declared sequelize in the branch?

Reply:
this is what the documentation shows how to do "greater than".